src/components/container/task_list.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { LEVELS } from '../../models/levels.enum';
+import { Task } from '../../models/task.class';
+import TaskComponent from '../pure/task';
+
+
+const TaskListComponent = () => {
+    const defaultTask = new Task('Example', 'Default description', false,LEVELS.NORMAL);
```

Comment:
Research about using eslint and prettier. 
Research about airbnb eslint config.

All this is for formatting the files automatically when you save them with your ide. So the identation, semicolons, etc are inserted automatically.